hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -438,14 +445,19 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
   );
 
   handlefsoNonfsoMenuChange = (e: any) => {
-    if (e.key === 'fso') {
-      openPrevKeyList =[""];
-      this.fetchOpenKeys(true, false, this.state.DEFAULT_LIMIT, "");
-    }
-    else {
-      openPrevKeyList = [""];
-      this.fetchOpenKeys(false, true, this.state.DEFAULT_LIMIT, "");
-    }
+    console.log("Non FSO handle", e.key);
```

Comment:
If this console.log is not required can we delete it? We should try to avoid logging in prod env.

Reply:
@devabhishekpal 

Removed console logs in latest commit.